src/System.Reflection.TypeExtensions/ref/System.Reflection.TypeExtensions.cs

```diff
@@ -65,19 +65,19 @@ public static partial class TypeExtensions
         public static System.Reflection.MemberInfo[] GetMembers(this System.Type type, System.Reflection.BindingFlags bindingAttr) { throw null; }
-        public static System.Reflection.MethodInfo GetMethod(this System.Type type, string name) { throw null; }
-        public static System.Reflection.MethodInfo GetMethod(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
-        public static System.Reflection.MethodInfo GetMethod(this System.Type type, string name, System.Type[] types) { throw null; }
+        public static System.Reflection.MethodInfo? GetMethod(this System.Type type, string name) { throw null; }
+        public static System.Reflection.MethodInfo? GetMethod(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
+        public static System.Reflection.MethodInfo? GetMethod(this System.Type type, string name, System.Type[] types) { throw null; }
         public static System.Reflection.MethodInfo[] GetMethods(this System.Type type) { throw null; }
         public static System.Reflection.MethodInfo[] GetMethods(this System.Type type, System.Reflection.BindingFlags bindingAttr) { throw null; }
-        public static System.Type GetNestedType(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
+        public static System.Type? GetNestedType(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
         public static System.Type[] GetNestedTypes(this System.Type type, System.Reflection.BindingFlags bindingAttr) { throw null; }
         public static System.Reflection.PropertyInfo[] GetProperties(this System.Type type) { throw null; }
         public static System.Reflection.PropertyInfo[] GetProperties(this System.Type type, System.Reflection.BindingFlags bindingAttr) { throw null; }
-        public static System.Reflection.PropertyInfo GetProperty(this System.Type type, string name) { throw null; }
-        public static System.Reflection.PropertyInfo GetProperty(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
-        public static System.Reflection.PropertyInfo GetProperty(this System.Type type, string name, System.Type returnType) { throw null; }
-        public static System.Reflection.PropertyInfo GetProperty(this System.Type type, string name, System.Type returnType, System.Type[] types) { throw null; }
+        public static System.Reflection.PropertyInfo? GetProperty(this System.Type type, string name) { throw null; }
+        public static System.Reflection.PropertyInfo? GetProperty(this System.Type type, string name, System.Reflection.BindingFlags bindingAttr) { throw null; }
+        public static System.Reflection.PropertyInfo? GetProperty(this System.Type type, string name, System.Type returnType) { throw null; }
```

Comment:
It seems like the ref assembly is still missing a `?` in returnType for both APIs...

Reply:
Also, for `IsAssignableFrom` (`System.Type c`) and `IsInstanceOfType` (`object o`)